openpype/hosts/flame/plugins/publish/extract_subset_resources.py

```diff
@@ -59,12 +60,7 @@ class ExtractSubsetResources(openpype.api.Extractor):
     export_presets_mapping = {}
 
     def process(self, instance):
-        if (
-            self.keep_original_representation
-            and "representations" not in instance.data
-            or not self.keep_original_representation
-        ):
-            instance.data["representations"] = []
+        self._make_representation_data(instance)
```

Comment:
I would simplify it to just 2 lines.
```suggestion
        if "representations" not in instance.data:
            instance.data["representations"] = []
```